src/Symfony/Component/Cache/Tests/Adapter/CouchbaseBucketAdapterTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Tests\Adapter;
+
+use Symfony\Component\Cache\Adapter\AbstractAdapter;
+use Symfony\Component\Cache\Adapter\CouchbaseBucketAdapter;
+
+class CouchbaseBucketAdapterTest extends AdapterTestCase
+{
+    /** @var \CouchbaseCluster */
+    protected static $client;
+
+    public static function setupBeforeClass()
+    {
+        if (!CouchbaseBucketAdapter::isSupported()) {
+            self::markTestSkipped('Extension couchbase >=2.6.0 required.');
```

Comment:
this should be replaced by an `@requires` annotation on the class

Reply:
Ok.